Do more [[likely]] / [[unlikely]] adoption in WebCore/

cdumez opened the pull request:
#### cbf24c0d5470db8030d01b052f7d963474fcfe59
<pre>
Do more [[likely]] / [[unlikely]] adoption in WebCore/
<a href="https://bugs.webkit.org/show_bug.cgi?id=292477">https://bugs.webkit.org/show_bug.cgi?id=292477</a>

Reviewed by Ryosuke Niwa.

Do more [[likely]] / [[unlikely]] adoption in WebCore/.

It is now part of the C++ language and has several benefits over
our macros. In particular, those annotations can be used for
`switch` cases and `else` statements.

* Source/WebCore/contentextensions/DFABytecodeInterpreter.cpp:
(WebCore::ContentExtensions::DFABytecodeInterpreter::interpret):
* Source/WebCore/crypto/SubtleCrypto.cpp:
(WebCore::normalizeCryptoAlgorithmParameters):
(WebCore::SubtleCrypto::unwrapKey):
* Source/WebCore/crypto/cocoa/CryptoKeyECMac.cpp:
(WebCore::CryptoKeyEC::platformExportRaw const):
(WebCore::CryptoKeyEC::platformAddFieldElements const):
(WebCore::CryptoKeyEC::platformExportSpki const):
(WebCore::CryptoKeyEC::platformExportPkcs8 const):
* Source/WebCore/crypto/gcrypt/CryptoKeyOKPGCrypt.cpp:
(WebCore::gcryptGenerateEd25519Keys):
(WebCore::gcryptGenerateX25519Keys):
* Source/WebCore/css/CSSSelector.cpp:
(WebCore::simpleSelectorSpecificity):
* Source/WebCore/css/CSSStyleProperties.cpp:
(WebCore::PropertySetCSSStyleProperties::setProperty):
* Source/WebCore/css/CSSStyleSheetObservableArray.cpp:
(WebCore::CSSStyleSheetObservableArray::setValueAt):
* Source/WebCore/css/CSSToStyleMap.cpp:
(WebCore::CSSToStyleMap::mapNinePieceImageQuad):
* Source/WebCore/css/DOMCSSPaintWorklet.cpp:
(WebCore::PaintWorklet::addModule):
* Source/WebCore/css/SelectorFilter.cpp:
(WebCore::SelectorFilter::pushParentInitializingIfNeeded):
* Source/WebCore/css/StyleAttributeMutationScope.h:
(WebCore::StyleAttributeMutationScope::StyleAttributeMutationScope):
* Source/WebCore/css/parser/CSSTokenizer.cpp:
(WebCore::CSSTokenizer::tryCreate):
(WebCore::CSSTokenizer::CSSTokenizer):
* Source/WebCore/dom/CharacterData.cpp:
(WebCore::CharacterData::parserAppendData):
* Source/WebCore/dom/ContainerNode.cpp:
(WebCore::ContainerNode::removeAllChildrenWithScriptAssertion):
(WebCore::ContainerNode::removeNodeWithScriptAssertion):
(WebCore::executeNodeInsertionWithScriptAssertion):
(WebCore::executeParserNodeInsertionIntoIsolatedTreeWithoutNotifyingParent):
(WebCore::ContainerNode::ensurePreInsertionValidityForPhantomDocumentFragment):
(WebCore::ContainerNode::removeBetween):
* Source/WebCore/dom/CustomElementReactionQueue.h:
(WebCore::CustomElementReactionStack::~CustomElementReactionStack):
* Source/WebCore/dom/CustomElementRegistry.h:
(WebCore::CustomElementRegistry::registryForElement):
(WebCore::CustomElementRegistry::registryForNodeOrTreeScope):
* Source/WebCore/dom/Document.cpp:
(WebCore::Document::setMarkupUnsafe):
(WebCore::createHTMLElementWithNameValidation):
(WebCore::Document::createElementForBindings):
(WebCore::Document::createElement):
(WebCore::validateCustomElementNameWithoutCheckingStandardElementNames):
(WebCore::Document::createElementNS):
(WebCore::Document::setTitle):
(WebCore::Document::write):
(WebCore::Document::parseQualifiedName):
(WebCore::Document::shouldMaskURLForBindingsInternal const):
(WebCore::Document::execCommand):
(WebCore::Document::queryCommandEnabled):
(WebCore::Document::queryCommandIndeterm):
(WebCore::Document::queryCommandState):
(WebCore::Document::queryCommandSupported):
(WebCore::Document::queryCommandValue):
(WebCore::Document::eventLoop):
(WebCore::Document::windowEventLoop):
(WebCore::Document::documentFragmentForInnerOuterHTML):
(WebCore::Document::compositeOperatorForBackgroundColor const):
* Source/WebCore/dom/DocumentInlines.h:
(WebCore::Document::invalidateAccessKeyCache):
(WebCore::Document::shouldMaskURLForBindings const):
(WebCore::Document::maskedURLForBindingsIfNeeded const):
* Source/WebCore/dom/Element.cpp:
(WebCore::Element::~Element):
(WebCore::Element::synchronizeAttribute const):
(WebCore::Element::notifyAttributeChanged):
(WebCore::getElementByIdIncludingDisconnected):
(WebCore::Element::didMoveToNewDocument):
(WebCore::Element::updateEffectiveTextDirectionIfNeeded):
(WebCore::Element::insertedIntoAncestor):
(WebCore::Element::removedFromAncestor):
(WebCore::Element::childrenChanged):
(WebCore::Element::removeAttribute):
(WebCore::Element::setOuterHTML):
(WebCore::Element::insertAdjacentHTML):
* Source/WebCore/dom/Element.h:
(WebCore::Element::disconnectFromIntersectionObservers):
(WebCore::Element::disconnectFromResizeObservers):
* Source/WebCore/dom/ElementInlines.h:
(WebCore::Element::hideNonce):
* Source/WebCore/dom/ElementRareData.h:
(WebCore::Element::removeShadowRoot):
* Source/WebCore/dom/EventContext.cpp:
(WebCore::EventContext::handleLocalEvents const):
* Source/WebCore/dom/EventListenerMap.cpp:
(WebCore::removeListenerFromVector):
* Source/WebCore/dom/EventPath.cpp:
(WebCore::EventPath::buildPath):
(WebCore::EventPath::setRelatedTarget):
(WebCore::EventPath::retargetTouch):
(WebCore::RelatedNodeRetargeter::RelatedNodeRetargeter):
* Source/WebCore/dom/EventTarget.cpp:
(WebCore::EventTarget::innerInvokeEventListeners):
* Source/WebCore/dom/ImageOverlay.cpp:
(WebCore::ImageOverlay::hasOverlay):
* Source/WebCore/dom/InternalObserverEvery.cpp:
* Source/WebCore/dom/InternalObserverFind.cpp:
* Source/WebCore/dom/InternalObserverForEach.cpp:
* Source/WebCore/dom/InternalObserverInspect.cpp:
* Source/WebCore/dom/InternalObserverLast.cpp:
* Source/WebCore/dom/InternalObserverReduce.cpp:
* Source/WebCore/dom/InternalObserverSome.cpp:
* Source/WebCore/dom/MessageEvent.cpp:
(WebCore::MessageEvent::create):
* Source/WebCore/dom/Microtasks.cpp:
(WebCore::MicrotaskQueue::runJSMicrotask):
(WebCore::MicrotaskQueue::performMicrotaskCheckpoint):
* Source/WebCore/dom/Node.cpp:
(WebCore::Node::convertNodesOrStringsIntoNodeVector):
(WebCore::Node::cloneNodeForBindings):
(WebCore::Node::moveTreeToNewScope):
(WebCore::Node::moveNodeToNewDocumentSlowCase):
* Source/WebCore/dom/Node.h:
(WebCore::EventTarget::ref):
(WebCore::EventTarget::deref):
* Source/WebCore/dom/SelectorQuery.cpp:
(WebCore::SelectorDataList::executeFastPathForIdSelector const):
(WebCore::filterRootById):
* Source/WebCore/dom/SlotAssignment.cpp:
(WebCore::nextSlotElementSkippingSubtree):
(WebCore::NamedSlotAssignment::didChangeSlot):
(WebCore::NamedSlotAssignment::assignSlots):
* Source/WebCore/dom/SlotAssignment.h:
(WebCore::ShadowRoot::resolveSlotsBeforeNodeInsertionOrRemoval):
(WebCore::ShadowRoot::willRemoveAllChildren):
(WebCore::ShadowRoot::didRemoveAllChildrenOfShadowHost):
(WebCore::ShadowRoot::didMutateTextNodesOfShadowHost):
(WebCore::ShadowRoot::hostChildElementDidChange):
* Source/WebCore/dom/Text.cpp:
(WebCore::Text::setDataAndUpdate):
* Source/WebCore/dom/TreeScope.cpp:
(WebCore::TreeScope::retargetToScope const):
(WebCore::TreeScope::ensureAdoptedStyleSheets):
* Source/WebCore/dom/WindowEventLoop.cpp:
(WebCore::WindowEventLoop::eventLoopForSecurityOrigin):
* Source/WebCore/editing/FrameSelection.cpp:
(WebCore::FrameSelection::nodeWillBeRemoved):
* Source/WebCore/editing/MarkupAccumulator.cpp:
(WebCore::appendCharactersReplacingEntitiesInternal):
* Source/WebCore/editing/TextIterator.cpp:
(WebCore::firstChild):
(WebCore::nextSibling):
(WebCore::nextNode):
(WebCore::isDescendantOf):
(WebCore::parentNodeOrShadowHost):
* Source/WebCore/editing/markup.cpp:
(WebCore::StyledMarkupAccumulator::appendStartTag):
(WebCore::StyledMarkupAccumulator::appendEndTag):
(WebCore::StyledMarkupAccumulator::traverseNodesForSerialization):
* Source/WebCore/html/CanvasBase.h:
(WebCore::CanvasBase::cssParserContext const):
* Source/WebCore/html/EmailInputType.cpp:
(WebCore::EmailInputType::sanitizeValue const):
* Source/WebCore/html/HTMLCanvasElement.cpp:
(WebCore::HTMLCanvasElement::getContext):
(WebCore::HTMLCanvasElement::paint):
* Source/WebCore/html/HTMLDocument.cpp:
(WebCore::HTMLDocument::namedItem):
* Source/WebCore/html/HTMLElement.cpp:
(WebCore::HTMLElement::nodeName const):
* Source/WebCore/html/HTMLFrameElementBase.cpp:
(WebCore::HTMLFrameElementBase::openURL):
* Source/WebCore/html/HTMLInputElement.cpp:
(WebCore::HTMLInputElement::updateType):
(WebCore::HTMLInputElement::placeholder const):
* Source/WebCore/html/HTMLMediaElement.cpp:
(WebCore::HTMLMediaElement::pauseAfterDetachedTask):
(WebCore::HTMLMediaElement::ensureMediaControls):
(WebCore::HTMLMediaElement::getCurrentMediaControlsStatus):
(WebCore::HTMLMediaElement::setShowingStats):
* Source/WebCore/html/HTMLSlotElement.cpp:
(WebCore::flattenAssignedNodes):
* Source/WebCore/html/HTMLTableElement.cpp:
(WebCore::HTMLTableElement::setTHead):
(WebCore::HTMLTableElement::setTFoot):
* Source/WebCore/html/InputType.cpp:
(WebCore::InputType::createIfDifferent):
* Source/WebCore/html/OffscreenCanvas.cpp:
(WebCore::OffscreenCanvas::getContext):
* Source/WebCore/html/TextFieldInputType.cpp:
(WebCore::limitLength):
(WebCore::TextFieldInputType::sanitizeValue const):
* Source/WebCore/html/canvas/CanvasPath.cpp:
(WebCore::CanvasPath::moveTo):
(WebCore::CanvasPath::lineTo):
(WebCore::CanvasPath::quadraticCurveTo):
(WebCore::CanvasPath::bezierCurveTo):
(WebCore::CanvasPath::arcTo):
(WebCore::CanvasPath::arc):
(WebCore::CanvasPath::ellipse):
(WebCore::CanvasPath::rect):
* Source/WebCore/html/canvas/CanvasRenderingContext2DBase.cpp:
(WebCore::CanvasRenderingContext2DBase::scale):
(WebCore::CanvasRenderingContext2DBase::rotate):
(WebCore::CanvasRenderingContext2DBase::translate):
(WebCore::CanvasRenderingContext2DBase::transform):
(WebCore::CanvasRenderingContext2DBase::fillInternal):
(WebCore::CanvasRenderingContext2DBase::strokeInternal):
(WebCore::CanvasRenderingContext2DBase::clipInternal):
(WebCore::CanvasRenderingContext2DBase::isPointInPathInternal):
(WebCore::CanvasRenderingContext2DBase::isPointInStrokeInternal):
(WebCore::CanvasRenderingContext2DBase::clearRect):
(WebCore::CanvasRenderingContext2DBase::fillRect):
(WebCore::CanvasRenderingContext2DBase::strokeRect):
(WebCore::CanvasRenderingContext2DBase::drawImage):
(WebCore::CanvasRenderingContext2DBase::didDraw):
(WebCore::CanvasRenderingContext2DBase::canDrawText):
* Source/WebCore/html/canvas/WebGLRenderingContextBase.cpp:
(WebCore::WebGLRenderingContextBase::prepareForDisplay):
* Source/WebCore/html/canvas/WebGLUtilities.cpp:
(WebCore::ScopedInspectorShaderProgramHighlight::shouldApply):
* Source/WebCore/html/canvas/WebGLUtilities.h:
(WebCore::ScopedInspectorShaderProgramHighlight::ScopedInspectorShaderProgramHighlight):
(WebCore::ScopedInspectorShaderProgramHighlight::~ScopedInspectorShaderProgramHighlight):

Canonical link: <a href="https://commits.webkit.org/294487@main">https://commits.webkit.org/294487@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/0d7610643ea6e459e71ec2896de20ddcb14721b6

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/102039 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/21707 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/12023 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/107198 "Built successfully") | [  ~~🛠 win~~](https://ews-build.webkit.org/#/builders/59/builds/52673 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/104079 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/22015 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/30214 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/77665 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/59/builds/52673 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/105046 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/17028 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/92127 "Passed tests") | [  ~~🧪 api-wpe~~](https://ews-build.webkit.org/#/builders/41/builds/58001 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/133/builds/16857 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/10152 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/52032 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/86695 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/10225 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/109571 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/29172 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/21491 "Passed tests") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/86635 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/29534 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/88331 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/86215 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/21931 "Built successfully and passed tests") | [  ~~🧪 vision-wk2~~](https://ews-build.webkit.org/#/builders/126/builds/31012 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/8731 "Passed tests") | [  ~~🛠 playstation~~](https://ews-build.webkit.org/#/builders/134/builds/23371 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/29100 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/34395 "Built successfully") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/28911 "Built successfully") | | | 
| | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/129/builds/32234 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/30470 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

rniwa commented on Source/WebCore/dom/Node.cpp:
```diff
@@ -2297,7 +2297,7 @@ void Node::moveTreeToNewScope(Node& root, TreeScope& oldScope, TreeScope& newSco
                 node.setEventTargetFlag(EventTargetFlag::HasBeenInUserAgentShadowTree);
             node.setTreeScope(newScope);
-            if (UNLIKELY(!node.hasRareData()))
+            if (!node.hasRareData()) [[unlikely]]
```
Wait a minute. This unlikely is wrong. It should be `[[likely]]` instead since we expect not having rare data to be the common case.

cdumez committed:
Do more [[likely]] / [[unlikely]] adoption in WebCore/
https://bugs.webkit.org/show_bug.cgi?id=292477

Reviewed by Ryosuke Niwa.

Do more [[likely]] / [[unlikely]] adoption in WebCore/.

It is now part of the C++ language and has several benefits over
our macros. In particular, those annotations can be used for
`switch` cases and `else` statements.

* Source/WebCore/contentextensions/DFABytecodeInterpreter.cpp:
(WebCore::ContentExtensions::DFABytecodeInterpreter::interpret):
* Source/WebCore/crypto/SubtleCrypto.cpp:
(WebCore::normalizeCryptoAlgorithmParameters):
(WebCore::SubtleCrypto::unwrapKey):
* Source/WebCore/crypto/cocoa/CryptoKeyECMac.cpp:
(WebCore::CryptoKeyEC::platformExportRaw const):
(WebCore::CryptoKeyEC::platformAddFieldElements const):
(WebCore::CryptoKeyEC::platformExportSpki const):
(WebCore::CryptoKeyEC::platformExportPkcs8 const):
* Source/WebCore/crypto/gcrypt/CryptoKeyOKPGCrypt.cpp:
(WebCore::gcryptGenerateEd25519Keys):
(WebCore::gcryptGenerateX25519Keys):
* Source/WebCore/css/CSSSelector.cpp:
(WebCore::simpleSelectorSpecificity):
* Source/WebCore/css/CSSStyleProperties.cpp:
(WebCore::PropertySetCSSStyleProperties::setProperty):
* Source/WebCore/css/CSSStyleSheetObservableArray.cpp:
(WebCore::CSSStyleSheetObservableArray::setValueAt):
* Source/WebCore/css/CSSToStyleMap.cpp:
(WebCore::CSSToStyleMap::mapNinePieceImageQuad):
* Source/WebCore/css/DOMCSSPaintWorklet.cpp:
(WebCore::PaintWorklet::addModule):
* Source/WebCore/css/SelectorFilter.cpp:
(WebCore::SelectorFilter::pushParentInitializingIfNeeded):
* Source/WebCore/css/StyleAttributeMutationScope.h:
(WebCore::StyleAttributeMutationScope::StyleAttributeMutationScope):
* Source/WebCore/css/parser/CSSTokenizer.cpp:
(WebCore::CSSTokenizer::tryCreate):
(WebCore::CSSTokenizer::CSSTokenizer):
* Source/WebCore/dom/CharacterData.cpp:
(WebCore::CharacterData::parserAppendData):
* Source/WebCore/dom/ContainerNode.cpp:
(WebCore::ContainerNode::removeAllChildrenWithScriptAssertion):
(WebCore::ContainerNode::removeNodeWithScriptAssertion):
(WebCore::executeNodeInsertionWithScriptAssertion):
(WebCore::executeParserNodeInsertionIntoIsolatedTreeWithoutNotifyingParent):
(WebCore::ContainerNode::ensurePreInsertionValidityForPhantomDocumentFragment):
(WebCore::ContainerNode::removeBetween):
* Source/WebCore/dom/CustomElementReactionQueue.h:
(WebCore::CustomElementReactionStack::~CustomElementReactionStack):
* Source/WebCore/dom/CustomElementRegistry.h:
(WebCore::CustomElementRegistry::registryForElement):
(WebCore::CustomElementRegistry::registryForNodeOrTreeScope):
* Source/WebCore/dom/Document.cpp:
(WebCore::Document::setMarkupUnsafe):
(WebCore::createHTMLElementWithNameValidation):
(WebCore::Document::createElementForBindings):
(WebCore::Document::createElement):
(WebCore::validateCustomElementNameWithoutCheckingStandardElementNames):
(WebCore::Document::createElementNS):
(WebCore::Document::setTitle):
(WebCore::Document::write):
(WebCore::Document::parseQualifiedName):
(WebCore::Document::shouldMaskURLForBindingsInternal const):
(WebCore::Document::execCommand):
(WebCore::Document::queryCommandEnabled):
(WebCore::Document::queryCommandIndeterm):
(WebCore::Document::queryCommandState):
(WebCore::Document::queryCommandSupported):
(WebCore::Document::queryCommandValue):
(WebCore::Document::eventLoop):
(WebCore::Document::windowEventLoop):
(WebCore::Document::documentFragmentForInnerOuterHTML):
(WebCore::Document::compositeOperatorForBackgroundColor const):
* Source/WebCore/dom/DocumentInlines.h:
(WebCore::Document::invalidateAccessKeyCache):
(WebCore::Document::shouldMaskURLForBindings const):
(WebCore::Document::maskedURLForBindingsIfNeeded const):
* Source/WebCore/dom/Element.cpp:
(WebCore::Element::~Element):
(WebCore::Element::synchronizeAttribute const):
(WebCore::Element::notifyAttributeChanged):
(WebCore::getElementByIdIncludingDisconnected):
(WebCore::Element::didMoveToNewDocument):
(WebCore::Element::updateEffectiveTextDirectionIfNeeded):
(WebCore::Element::insertedIntoAncestor):
(WebCore::Element::removedFromAncestor):
(WebCore::Element::childrenChanged):
(WebCore::Element::removeAttribute):
(WebCore::Element::setOuterHTML):
(WebCore::Element::insertAdjacentHTML):
* Source/WebCore/dom/Element.h:
(WebCore::Element::disconnectFromIntersectionObservers):
(WebCore::Element::disconnectFromResizeObservers):
* Source/WebCore/dom/ElementInlines.h:
(WebCore::Element::hideNonce):
* Source/WebCore/dom/ElementRareData.h:
(WebCore::Element::removeShadowRoot):
* Source/WebCore/dom/EventContext.cpp:
(WebCore::EventContext::handleLocalEvents const):
* Source/WebCore/dom/EventListenerMap.cpp:
(WebCore::removeListenerFromVector):
* Source/WebCore/dom/EventPath.cpp:
(WebCore::EventPath::buildPath):
(WebCore::EventPath::setRelatedTarget):
(WebCore::EventPath::retargetTouch):
(WebCore::RelatedNodeRetargeter::RelatedNodeRetargeter):
* Source/WebCore/dom/EventTarget.cpp:
(WebCore::EventTarget::innerInvokeEventListeners):
* Source/WebCore/dom/ImageOverlay.cpp:
(WebCore::ImageOverlay::hasOverlay):
* Source/WebCore/dom/InternalObserverEvery.cpp:
* Source/WebCore/dom/InternalObserverFind.cpp:
* Source/WebCore/dom/InternalObserverForEach.cpp:
* Source/WebCore/dom/InternalObserverInspect.cpp:
* Source/WebCore/dom/InternalObserverLast.cpp:
* Source/WebCore/dom/InternalObserverReduce.cpp:
* Source/WebCore/dom/InternalObserverSome.cpp:
* Source/WebCore/dom/MessageEvent.cpp:
(WebCore::MessageEvent::create):
* Source/WebCore/dom/Microtasks.cpp:
(WebCore::MicrotaskQueue::runJSMicrotask):
(WebCore::MicrotaskQueue::performMicrotaskCheckpoint):
* Source/WebCore/dom/Node.cpp:
(WebCore::Node::convertNodesOrStringsIntoNodeVector):
(WebCore::Node::cloneNodeForBindings):
(WebCore::Node::moveTreeToNewScope):
(WebCore::Node::moveNodeToNewDocumentSlowCase):
* Source/WebCore/dom/Node.h:
(WebCore::EventTarget::ref):
(WebCore::EventTarget::deref):
* Source/WebCore/dom/SelectorQuery.cpp:
(WebCore::SelectorDataList::executeFastPathForIdSelector const):
(WebCore::filterRootById):
* Source/WebCore/dom/SlotAssignment.cpp:
(WebCore::nextSlotElementSkippingSubtree):
(WebCore::NamedSlotAssignment::didChangeSlot):
(WebCore::NamedSlotAssignment::assignSlots):
* Source/WebCore/dom/SlotAssignment.h:
(WebCore::ShadowRoot::resolveSlotsBeforeNodeInsertionOrRemoval):
(WebCore::ShadowRoot::willRemoveAllChildren):
(WebCore::ShadowRoot::didRemoveAllChildrenOfShadowHost):
(WebCore::ShadowRoot::didMutateTextNodesOfShadowHost):
(WebCore::ShadowRoot::hostChildElementDidChange):
* Source/WebCore/dom/Text.cpp:
(WebCore::Text::setDataAndUpdate):
* Source/WebCore/dom/TreeScope.cpp:
(WebCore::TreeScope::retargetToScope const):
(WebCore::TreeScope::ensureAdoptedStyleSheets):
* Source/WebCore/dom/WindowEventLoop.cpp:
(WebCore::WindowEventLoop::eventLoopForSecurityOrigin):
* Source/WebCore/editing/FrameSelection.cpp:
(WebCore::FrameSelection::nodeWillBeRemoved):
* Source/WebCore/editing/MarkupAccumulator.cpp:
(WebCore::appendCharactersReplacingEntitiesInternal):
* Source/WebCore/editing/TextIterator.cpp:
(WebCore::firstChild):
(WebCore::nextSibling):
(WebCore::nextNode):
(WebCore::isDescendantOf):
(WebCore::parentNodeOrShadowHost):
* Source/WebCore/editing/markup.cpp:
(WebCore::StyledMarkupAccumulator::appendStartTag):
(WebCore::StyledMarkupAccumulator::appendEndTag):
(WebCore::StyledMarkupAccumulator::traverseNodesForSerialization):
* Source/WebCore/html/CanvasBase.h:
(WebCore::CanvasBase::cssParserContext const):
* Source/WebCore/html/EmailInputType.cpp:
(WebCore::EmailInputType::sanitizeValue const):
* Source/WebCore/html/HTMLCanvasElement.cpp:
(WebCore::HTMLCanvasElement::getContext):
(WebCore::HTMLCanvasElement::paint):
* Source/WebCore/html/HTMLDocument.cpp:
(WebCore::HTMLDocument::namedItem):
* Source/WebCore/html/HTMLElement.cpp:
(WebCore::HTMLElement::nodeName const):
* Source/WebCore/html/HTMLFrameElementBase.cpp:
(WebCore::HTMLFrameElementBase::openURL):
* Source/WebCore/html/HTMLInputElement.cpp:
(WebCore::HTMLInputElement::updateType):
(WebCore::HTMLInputElement::placeholder const):
* Source/WebCore/html/HTMLMediaElement.cpp:
(WebCore::HTMLMediaElement::pauseAfterDetachedTask):
(WebCore::HTMLMediaElement::ensureMediaControls):
(WebCore::HTMLMediaElement::getCurrentMediaControlsStatus):
(WebCore::HTMLMediaElement::setShowingStats):
* Source/WebCore/html/HTMLSlotElement.cpp:
(WebCore::flattenAssignedNodes):
* Source/WebCore/html/HTMLTableElement.cpp:
(WebCore::HTMLTableElement::setTHead):
(WebCore::HTMLTableElement::setTFoot):
* Source/WebCore/html/InputType.cpp:
(WebCore::InputType::createIfDifferent):
* Source/WebCore/html/OffscreenCanvas.cpp:
(WebCore::OffscreenCanvas::getContext):
* Source/WebCore/html/TextFieldInputType.cpp:
(WebCore::limitLength):
(WebCore::TextFieldInputType::sanitizeValue const):
* Source/WebCore/html/canvas/CanvasPath.cpp:
(WebCore::CanvasPath::moveTo):
(WebCore::CanvasPath::lineTo):
(WebCore::CanvasPath::quadraticCurveTo):
(WebCore::CanvasPath::bezierCurveTo):
(WebCore::CanvasPath::arcTo):
(WebCore::CanvasPath::arc):
(WebCore::CanvasPath::ellipse):
(WebCore::CanvasPath::rect):
* Source/WebCore/html/canvas/CanvasRenderingContext2DBase.cpp:
(WebCore::CanvasRenderingContext2DBase::scale):
(WebCore::CanvasRenderingContext2DBase::rotate):
(WebCore::CanvasRenderingContext2DBase::translate):
(WebCore::CanvasRenderingContext2DBase::transform):
(WebCore::CanvasRenderingContext2DBase::fillInternal):
(WebCore::CanvasRenderingContext2DBase::strokeInternal):
(WebCore::CanvasRenderingContext2DBase::clipInternal):
(WebCore::CanvasRenderingContext2DBase::isPointInPathInternal):
(WebCore::CanvasRenderingContext2DBase::isPointInStrokeInternal):
(WebCore::CanvasRenderingContext2DBase::clearRect):
(WebCore::CanvasRenderingContext2DBase::fillRect):
(WebCore::CanvasRenderingContext2DBase::strokeRect):
(WebCore::CanvasRenderingContext2DBase::drawImage):
(WebCore::CanvasRenderingContext2DBase::didDraw):
(WebCore::CanvasRenderingContext2DBase::canDrawText):
* Source/WebCore/html/canvas/WebGLRenderingContextBase.cpp:
(WebCore::WebGLRenderingContextBase::prepareForDisplay):
* Source/WebCore/html/canvas/WebGLUtilities.cpp:
(WebCore::ScopedInspectorShaderProgramHighlight::shouldApply):
* Source/WebCore/html/canvas/WebGLUtilities.h:
(WebCore::ScopedInspectorShaderProgramHighlight::ScopedInspectorShaderProgramHighlight):
(WebCore::ScopedInspectorShaderProgramHighlight::~ScopedInspectorShaderProgramHighlight):

Canonical link: https://commits.webkit.org/294487@main
